boards/arm/mec1501modular_assy6885/mec1501modular_assy6885_defconfig

```diff
@@ -7,8 +7,9 @@
 CONFIG_ARM=y
 CONFIG_SOC_MEC1501_HSZ=y
 CONFIG_SOC_SERIES_MEC1501X=y
+CONFIG_SOC_MEC1501_VTR3_1_8V=y
 CONFIG_BOARD_MEC1501MODULAR_ASSY6885=y
-CONFIG_RTOS_TIMER=n
+CONFIG_RTOS_TIMER=y
```

Comment:
This change to RTOS_TIMER isn't related.  Should be at least a separate commit if not PR.

Reply:
Removed this from this PR, will enable RTOS driver into another PR that enables other drivers as well.